packages/experiments/src/components/Shimmer/Shimmer.types.ts

```diff
@@ -123,7 +132,7 @@ export interface IShimmerStyles {
 }
 
 export const enum ShimmerElementType {
-  RECTANGLE = 'line',
+  LINE = 'line',
```

Comment:
Why are these enum values TITLECASED? They should just be camelCased .

Reply:
Got it. Will definitely change them in my next PR. Thanks!